lesson_13/edmond_sabou/home1_les13.py

```diff
@@ -0,0 +1,63 @@
+from math import sqrt
+
+
+class Perimeter:
+
+    def perimeter(self):
+        class_name = type(self).__name__
+        return f'The perimeter for {class_name} is {sum(self.sides)}.\n'
+
+
+class Polygon(Perimeter):
```

Comment:
Placing the Mixin here will make the perimeter available for all shapes that inherit from Polygon.

Where we can move the mixin to have the perimeter method only on instances of Triangles?

Reply:
moved the Mixin just above the Triangles class, moved every other class above the Mixin class